app/models/article.rb

```diff
@@ -218,14 +220,16 @@ class Article < ApplicationRecord
     case tags
     when String
       cached_tagged_with(tags)
+    when Symbol
+      cached_tagged_with(tags)
```

Comment:
I don't know why the suggestion feature works here but didn't work for me in the one above.

```suggestion
    when String, Symbol
      cached_tagged_with(tags)
```